docs/how-to/rccl-usage-tips.rst

```diff
@@ -82,22 +82,174 @@ set the HSA environment variable as follows:
-However, if your workload warrants using fewer than 8 MI300X accelerators on a system,
-you can set the run-time variable ``NCCL_MIN_NCHANNELS`` to increase the number of channels. For example:
+This section outlines ways to improve RCCL performance on MI300X systems,
+including guidelines for systems with fewer than eight GPUs and the most efficient
+GPU partition modes.
+
+Configuration with fewer than eight GPUs
+----------------------------------------
+
+On a system with eight MI300X accelerators, each pair of accelerators is connected
+with dedicated Infinity Fabric™ links in a fully connected topology. For
+collective operations, this can achieve good performance when all eight accelerators
+(and all Infinity Fabric links) are used. When fewer than eight GPUs are used,
+however, this can only achieve a fraction of the potential bandwidth on the
+system. However, if your workload warrants using fewer than eight MI300X accelerators
+on a system, you can set the run-time variable ``NCCL_MIN_NCHANNELS`` to
+increase the number of channels. For example:
 
 .. code-block:: shell
 
    export NCCL_MIN_NCHANNELS=32
 
 Increasing the number of channels can benefit performance, but it also increases
 GPU utilization for collective operations.
-Additionally, RCCL pre-defines a higher number of channels when only 2 or
-4 accelerators are in use on a 8\*MI300X system. In this situation, RCCL uses 32 channels with two MI300X accelerators
-and 24 channels for four MI300X accelerators.
\ No newline at end of file
+Additionally, RCCL pre-defines a higher number of channels when only two or four
+accelerators are in use on a 8\*MI300X system. In this situation, RCCL uses 32
+channels with two MI300X accelerators and 24 channels for four MI300X
+accelerators.
+
+.. _nps4_cpx_mi300_rccl:
+
+NPS4 and CPX partition modes
+----------------------------
+
+There are multiple spatial partitioning modes available, such as SPX, DPX, TPX,
+QPX, and CPX. These modes are referred to as compute partition modes.
+
+Similarly, there are different memory partition modes available, such as NPS1,
+NPS2, and NPS4. A compute partition mode can only be combined with specific
+memory partition modes.
+
+- Compute partition modes 
+
+  - SPX - Single partition mode.
+  
+  - DPX - Dual partition mode or two equal spatial partitions.
+
+  - TPX - Triple partition mode or three equal spatial partitions.
+
+  - QPX - Quad partition mode or four equal spatial partitions.
+
+  - CPX - Core partition mode. In this mode, each compute partition
+    mode only has a single core cluster.
+  
+- Memory partition modes 
+
+  - NPS1 - One memory partition/NUMA domain per device.
+
+  - NPS2 - Two memory partitions/NUMA domains per device.
+
+  - NPS4 - Four memory partitions/NUMA domains per device.
+
+For more information, see :doc:`Dynamically change GPU partition modes <rocm:how-to/dynamic_partition_modes>`
+page. The MI300 can be set to CPX mode using the ``rocm-smi`` commands below.
```

Comment:
An MI300 GPU can be set to CPX mode ... seems to be a better wording for the sentence.

Reply:
Updated.